### ISSUE TITLE
/whois fails if I have a status set through the webapp

### ISSUE THREAD

**Issue by apetresc** [wee-slack/wee-slack#624]
Under normal circumstances, I can `/whois` myself just fine:

```
08:16:39 | [apetresc]: Adrian Petrescu
08:16:39 | [apetresc]: Real name: *redacted*
08:16:39 | [apetresc]: Title: *redacted*
08:16:39 | [apetresc]: Email: *redacted*
08:16:39 | [apetresc]: Phone: *redacted*
08:16:58 | [apetresc]: *redacted*
```

But if I use the Slack web app to set a status (like, say, ":house_with_garden: WFH"), then a `/whois` on myself fails with:
```
08:13:28 =!= | irc: command "whois" must be executed on irc buffer (server or channel)
08:16:58     | python: stdout/stderr (slack): Traceback (most recent call last):
08:16:58     | python: stdout/stderr (slack):   File "/home/apetresc/.weechat/python/autoload/wee_slack.py", line 103, in wrapper
08:16:58     | python: stdout/stderr (slack):     return f(data, current_buffer, *args, **kwargs)
08:16:58     | python: stdout/stderr (slack):   File "/home/apetresc/.weechat/python/autoload/wee_slack.py", line 126, in wrapper
08:16:58     | python: stdout/stderr (slack):     return f(*decode_from_utf8(args), **decode_from_utf8(kwargs))
08:16:58     | python: stdout/stderr (slack):   File "/home/apetresc/.weechat/python/autoload/wee_slack.py", line 3201, in whois_command_cb
08:16:58     | python: stdout/stderr (slack):     team.buffer_prnt("[{}]: {} {}".format(user, u.profile.status_emoji, u.profile.status_text))
08:16:58     | python: stdout/stderr (slack): AttributeError: 'dict' object has no attribute 'status_emoji'
08:16:58 =!= | python: error in function "whois_command_cb"
```

If I then clear my status and `/whois` myself again, it goes back to working.

Strangely, if I `/whois` _other_ people who also happen to have an emoji status set, it works fine for them. (It doesn't show me their status or anything, but it does at least not error out).

Let me know if there's any other details I can provide! :)

**Comment by auscompgeek** [comment]
Huh, the part about /whois on other people working is odd.

I encountered this bug myself the other day. I'll probably send a PR tomorrow if I get the chance.

**Comment by trygveaa** [comment]
Fixed with #651.
